app/assets/javascripts/application.js

```diff
@@ -17,4 +17,4 @@
 //= require chosen-jquery
 //= require_tree .
 
-$(function(){ $(document).foundation(); });
+$(document).foundation();
```

Comment:
? Why

Reply:
chrome browser was complaining, so this seemed to fix it https://github.com/zurb/foundation-rails#add-foundation-to-your-js

Reply:
That's interesting, maybe things have changed recently. What was Chrome complaining about. I can't see any messages...

Reply:
it wasn't complaining until i started trying the tooltip, so maybe there wasn't any foundation js being used yet. the complaint was something about finding foundation js more than once.